apps/condo/domains/user/schema/User.js

```diff
@@ -374,6 +385,23 @@ const User = new GQLListSchema('User', {
             ) {
                 await updateEmployeesRelatedToUser(context, updatedItem)
             }
+
+            if (updatedItem.type === STAFF) {
+                const isEmailChanged = updatedItem.email !== (existingItem ? existingItem.email : null)
+                const isEmailVerifiedChanged = updatedItem.isEmailVerified !== (existingItem ? existingItem.isEmailVerified : null)
+                const isMarketingConsentChanged = updatedItem.hasMarketingConsent !== (existingItem ? existingItem.hasMarketingConsent : null)
+
+                if (isEmailChanged || isEmailVerifiedChanged || isMarketingConsentChanged) {
```

Comment:
Should this be called if the user removes the email? This is a valid operation in our api

Reply:
Also i think `isEmailVerified` should be reset when user changes his email. (out of this pr)